@here/olp-sdk-dataservice-write/lib/client/VersionedLayerClient.ts

```diff
@@ -18,7 +18,8 @@
  */
 
 import { HRN, OlpClientSettings, RequestFactory } from "@here/olp-sdk-core";
-import { MetadataApi } from "@here/olp-sdk-dataservice-api";
+import { MetadataApi, PublishApi } from "@here/olp-sdk-dataservice-api";
+import { StartBatchRequest } from "../model";
```

Comment:
I think we should follow the dataservice-read pattern and place the request structures in the same folder as the clients.

Reply:
moved